ibridges/__main__.py

```diff
@@ -131,88 +132,222 @@ def main() -> None:  #pylint: disable=too-many-branches
         ibridges_meta_add()
     elif subcommand in ["meta-del", "meta-rm"]:
         ibridges_meta_del()
+    elif subcommand == "alias":
+        ibridges_alias()
+    elif subcommand == "cd":
+        ibridges_cd()
+    elif subcommand == "pwd":
+        ibridges_pwd()
     else:
         print(f"Invalid subcommand ({subcommand}). For help see ibridges --help")
         sys.exit(1)
 
-def _get_ibridges_conf(ienv_path) -> dict:
+def ibridges_cd():
+    """Change current working collection to another path."""
+    parser = argparse.ArgumentParser(
+        prog="ibridges cd", description="Change collection to another path."
+    )
+    parser.add_argument(
+        "collection",
+        help="Change collection in iRODS.",
+        type=str,
+        default="~",
+        nargs="?"
+    )
+    args = parser.parse_args()
+
+    ibridges_conf = _get_ibridges_conf()
+    with _cli_auth(ibridges_conf["cur_ienv"]) as session:
+        new_ipath = IrodsPath(session, args.collection)
+        if not new_ipath.collection_exists():
+            print(f"Collection {new_ipath} does not exist.")
+            sys.exit(125)
+        entry = _get_ienv_entry(ibridges_conf["cur_ienv"], ibridges_conf)
+        entry[1]["cwd"] = str(new_ipath)
+        _set_ibridges_conf(ibridges_conf)
+
+
+def ibridges_pwd():
+    """Print current working directory."""
+    ibridges_conf = _get_ibridges_conf()
+    entry = _get_ienv_entry(ibridges_conf["cur_ienv"], ibridges_conf)
+    if "cwd" in entry[1]:
+        cwd = entry[1]["cwd"]
+    else:
+        with open(entry[0], "r", encoding="utf-8") as handle:
+            cwd = json.load(handle).get("irods_home", "unknown")
+    print(cwd)
+
+def ibridges_alias():
+    """Print existing aliases or create new ones."""
+    parser = argparse.ArgumentParser(
+        prog="ibridges alias",
+        description="Create and list aliases for your iRODS environment files."
+    )
+    parser.add_argument(
+        "alias",
+        help="The new alias to be created",
+        type=str,
+        default=None,
+        nargs="?",
+    )
+    parser.add_argument(
+        "env_path",
+        help="iRODS environment path.",
+        type=Path,
+        default=None,
+        nargs="?"
+    )
+    parser.add_argument(
+        "--delete", "-d",
+        help="Delete the alias.",
+        action="store_true",
+    )
+    args = parser.parse_args()
+    ibridges_conf = _get_ibridges_conf()
+    if args.alias is None:
+        for ienv_path, entry in ibridges_conf["servers"].items():
+            prefix = " "
+            if ibridges_conf["cur_ienv"] in (entry.get("alias", None), ienv_path):
+                prefix = "*"
+            cur_alias = entry.get("alias", "[no alias]")
+            print(f"{prefix} {cur_alias} -> {ienv_path}")
+        return
+
+    if args.delete:
```

Comment:
I think this section needs a tiny little bit of commenting how the logic functions. I think I understand it now, but it was not obvious.